config/config.go

```diff
@@ -7,13 +7,15 @@ import (
 	"github.com/libp2p/go-libp2p-core/connmgr"
 	"github.com/libp2p/go-libp2p-core/crypto"
 	"github.com/libp2p/go-libp2p-core/host"
+	coreit "github.com/libp2p/go-libp2p-core/introspection"
```

Comment:
Let's shorten this package name to `introspect`. And you can remove the `coreit` alias.

Reply:
Done